appgate/resource_appgate_policy.go

```diff
@@ -284,6 +300,11 @@ func basePolicyClientAttributes() map[string]*schema.Schema {
 						Required: true,
 						Elem:     &schema.Schema{Type: schema.TypeString},
 					},
+					"force": {
```

Comment:
this attribute is only used in the new policy type `Stop`. I suggest adding a new resource, called `appgatesdp_stop_policy`. However we could do that in a separate pull request.